### PR TITLE
Allow for moving clients between screens

### LINKED ISSUE #104: multi screens shortcuts
the shortcut to move a window to another screen does not work, any ideas?

### PR DESCRIPTION
**Pull request by laloch**
Fixes #104

@faho If you only use one display and you want to test multiscreen, you can create a virtual screen like this:
```
xrandr --newmode "1624x768_60.00"  101.25  1624 1704 1872 2120  768 771 781 798 -hsync +vsync
xrandr --addmode VIRTUAL1 1624x768_60.00
xrandr --output VIRTUAL1 --mode 1624x768_60.00
x11vnc -display :0 -clip xinerama1 -usepw -xrandr -forever -nonc -noxdamage -repeat -multiptr
```
and connect to it from a VNC client on your smart phone. If you want/need different aspect ratio for your display, you can generate custom modeline using `cvt` command.

### CODE REVIEW

File: contents/code/tilelist.js [hunk 1]
```diff
@@ -187,6 +187,12 @@ TileList.prototype.connectSignals = function(client) {
             tile.onClientDesktopChanged(client);
         }
     });
+    client.screenChanged.connect(function() {
```
Comment (faho) [inline contents/code/tilelist.js]: That easy?

My main concern, and I've brought this up with the KWin folks before, is ordering of the signals.

Does screenChanged always fire before e.g. moveResized?

Comment (laloch) [reply contents/code/tilelist.js]: `screenChanged` fires when moving finishes, just before `clientFinishUserMovedResized`. I was also concerned about the ordering. I did all kinds of things to my windows - moving between screens using window menu, dragging the whole windows, even resizing over the edge of the screen and so far everything seems to work just fine.

Comment (faho) [reply contents/code/tilelist.js]: Okay, so let's merge it and see what happens!

**Comment by laloch** [comment]
Thanks for the review!

**Pull request merged**